docs/lang/static.md

```diff
@@ -24,7 +24,7 @@ seq {
 }
 ```
 
-A simple trick to achieve this is adding an empty group with `@static(n)` attribute on it:
+A simple way to achieve this is adding an empty group with `@static(n)` attribute on it:
 ```
 cell {
   r = @std_reg(0);
```

Comment:
In the paper, we say that the `delay_n` group can just be empty. Was that an oversimplication, or is this treatment a little overkill? Can I just lose this register and the delay group's body?

Reply:
I think so! This example is old and meant to be used with the `@static` annotation. The source of truth is always the compiler if you're in doubt, try compiling the relevant program and see what happens!

Reply:
Thanks! This may just get deleted anyway; please see the updated top-level summary

Reply:
Yeah, let's nix this for now! I think the trick is useful but maybe we can find the right place to put it after we have the whole docs done.